### PR TITLE
Optional normalization for generic_cell_clustering.ipynb

### LINKED ISSUE #1000: Make 99.9 normalization optional on function cell_som_clustering.train_cell_som
**Is your feature request related to a problem? Please describe.**
I want to use the [templates/generic_cell_clustering.ipynb](https://github.com/angelolab/ark-analysis/blob/main/templates/generic_cell_clustering.ipynb) on data from the cell classification model. @ngreenwald mentioned that I should run the SOM clustering without 99.9 percentile normalization on this data, but the function `train_cell_som` and the downstream class `CellSOMCluster` do not allow me to turn off normalization.

**Describe the solution you'd like**
Add a flag to [cluster_helpers.CellSOMCluster](https://github.com/angelolab/ark-analysis/blob/main/src/ark/phenotyping/cluster_helpers.py) to the init called `normalize` with default `True` and add control flow to the init to make `self.train_data = self.normalize_data(self.train_data)` optional.

Add a flag to [cell_som_clustering.train_cell_som](https://github.com/angelolab/ark-analysis/blob/main/src/ark/phenotyping/cell_som_clustering.py) called `normalize` with default `True` and provide this argument to `cluster_helpers.CellSOMCluster`.

**Describe alternatives you've considered**
None

**Additional context**
None

### PR DESCRIPTION
**Pull request by camisowers**
**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**
Closes #1000. Make the 99.9 percentile normalization in cell clustering optional.

**How did you implement your changes**
Adds a `normalize` flag to `train_cell_SOM` and in the CellSOMCluster class which is defaulted to True.
Changing this arg to False will not conduct 99.9 percentile normalization on the data before cell clustering.

**Remaining issues**
N/A

### CODE REVIEW

**Review by alex-l-kong — CHANGES_REQUESTED** [review]
One comment about notebook documentatio

File: templates/generic_cell_clustering.ipynb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
```
Comment (alex-l-kong) [inline templates/generic_cell_clustering.ipynb:1]: <p>It's better to allow the user to set <code>normalize</code> in section 1. Either put it in its own cell or add it to one of the other param setting cells.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1027/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

**Review by srivarra — APPROVED** [review]

**Review by alex-l-kong — APPROVED** [review]

File: templates/generic_cell_clustering.ipynb (hunk 1 above)
Comment (cliu72) [inline templates/generic_cell_clustering.ipynb:1]: <p>I think just a small typo - <code>cell_som_cluster_cols</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1027/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

**Review by cliu72 — COMMENTED** [review]
Looks good to me, just one small typo in the jupyter notebook

**Review by cliu72 — APPROVED** [review]

File: templates/generic_cell_clustering.ipynb [hunk 2]
```diff
@@ -229,27 +221,46 @@
    ]
   },
   {
-   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**99.9% Normalization**\n",
+    "\n",
+    "Whether to normalize each of the `cell_som_cluster_cols` by their 99.9% value prior to training the data. Set to `False` to skip this normalization step."
```
Comment (JLrumberger) [inline templates/generic_cell_clustering.ipynb:229]: Text could be changed from "training the data" to "training the SOM".

**Pull request merged**